### PR TITLE
Juarezds

### PR DESCRIPTION
**Pull request by iamdez99**

### CODE REVIEW

**Review by shomiruddin94 — APPROVED** [review]

File: src/main/java/com/nci/automation/web/CommonUtils.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import java.text.SimpleDateFormat;
 import java.time.Duration;
 import java.util.*;
+
```
Comment (hamidnesar) [inline src/main/java/com/nci/automation/web/CommonUtils.java:12]: I don't believe an empty line would be needed

File: src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java [hunk 2]
```diff
@@ -3,7 +3,11 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Set;
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java:6]: Empty line

File: src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java [hunk 3]
```diff
@@ -368,7 +335,7 @@ public void enters_proband_home_phone_number_cell_phone_number_work_phone_number
         rasopathyQuestionnairePage.pleaseListCellPhoneNumberTextBox.sendKeys(foreigncellPhoneNumber);
         rasopathyQuestionnairePage.pleaseListWorkPhoneNumberTextBox.sendKeys(foreignWorkPhoneNumber);
         rasopathyQuestionnairePage.pleaseSelectCellPhonePreferredContactCheckBox.click();
-        
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java:338]: Do we need these empty lines almost for every step in this class: MyRASScreenerSubmissionSteps

**Review by hamidnesar — COMMENTED** [review]
I noticed a few empty lines around Imports and do we need these empty lines almost for every step in this class: MyRASScreenerSubmissionSteps

**Review by georgeb79 — APPROVED** [review]

**Review by andreyeve — APPROVED** [review]
Reviewed all and approved.

File: src/test/java/ServiceNow/CHARMS/Pages/ParticipantDetailsPage.java [hunk 4]
```diff
@@ -21,6 +21,26 @@ public WebElement dynamicRecordButtonLocator(String text) {
     @FindBy(xpath = "//*[contains(text(),'Open Record')]")
     public WebElement openRecordButton;
 
+    /* FIRST NAME RECORD PREVIEW FIELD */
```
Comment (mzuhoor) [inline src/test/java/ServiceNow/CHARMS/Pages/ParticipantDetailsPage.java:24]: Should the comments for the Elements be all in upper case letters? are we following any code standard regarding the naming convention, comments, spacing, and indentation? I have been reviewing PRs for the past few days and I have noticed lots of issues regarding the comments, spacing, and indentation.

File: src/main/java/com/nci/automation/web/CommonUtils.java (hunk 1 above)
Comment (iamdez99) [reply src/main/java/com/nci/automation/web/CommonUtils.java:12]: fixed

File: src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java (hunk 2 above)
Comment (iamdez99) [reply src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java:6]: fixed

File: src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java (hunk 3 above)
Comment (iamdez99) [reply src/test/java/ServiceNow/CHARMS/Steps/MyRASScreenerSubmissionSteps.java:338]: Those steps will be removed soon

File: src/test/java/appsCommon/DynamicLocators.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package appsCommon;
+
+import com.nci.automation.web.WebDriverUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class DynamicLocators {
+    /**
+     * THIS CLASS CONTAINS DYNAMIC LOCATORS
```
Comment (mzuhoor) [inline src/test/java/appsCommon/DynamicLocators.java:9]: Would it be better to have the comment right above the Class?
/**
     * THIS CLASS CONTAINS DYNAMIC LOCATORS
     * /
  public class DynamicLocators {

**Review by hamidnesar — APPROVED** [review]

File: src/test/java/appsCommon/DynamicLocators.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+     */
+
+    /***
```
Comment (mzuhoor) [inline src/test/java/appsCommon/DynamicLocators.java:12]: The extra * can be removed

File: src/test/java/appsCommon/DynamicLocators.java (hunk 5 above)
Comment (iamdez99) [reply src/test/java/appsCommon/DynamicLocators.java:9]: No, its needs to be inside the class

File: src/test/java/appsCommon/DynamicLocators.java (hunk 6 above)
Comment (iamdez99) [reply src/test/java/appsCommon/DynamicLocators.java:12]: This actually does not look like this in intelliJ

File: src/test/java/appsCommon/DynamicLocators.java (hunk 5 above)
Comment (mzuhoor) [reply src/test/java/appsCommon/DynamicLocators.java:9]: Oh ok, I didn't know that before. I thought the comments always go above the line regardless of the class, method, or line.

**Review by mzuhoor — APPROVED** [review]

**Review by mzuhoor — APPROVED** [review]

**Pull request merged**